lib/puppet/rest/client.rb

```diff
@@ -38,8 +38,8 @@ def initialize(ssl_store: OpenSSL::X509::Store.new,
       @dns_resolver = Puppet::Network::Resolver.new
     end
 
-    # Make a GET request to the specified URL with the specified params.
-    # @param [String] url the full path to query
+    # Make a GET request to the specified endpoint with the specified params.
+    # @param [String] endpoint the endpoint of the configured API to query
     # @param [Hash] query any URL params to add to send to the endpoint
```

Comment:
Were these docs changed back to "endpoint" intentionally? I updated the wording in my last PR to make it clear that we are no longer pre-calculating the base URL. The ability to configure the client with a base URL is called out as a feature of the HTTPClient library, so I thought it was worth making that distinction here.

Reply:
I probably overwrote your change during a rebase with master, as I don't have any memory of changing this to 'endpoint'. I'll make sure that gets corrected.